go.mod

```diff
@@ -6,24 +6,22 @@ require (
 	cloud.google.com/go v0.56.0
 	cloud.google.com/go/pubsub v1.3.1
 	cloud.google.com/go/storage v1.6.0
-	github.com/crossplane/crossplane v0.11.0
-	github.com/crossplane/crossplane-runtime v0.9.1-0.20200827134951-29150890d4c1
-	github.com/crossplane/crossplane-tools v0.0.0-20200827141855-f51a6598f2bc
+	github.com/crossplane/crossplane v0.13.0-rc.0.20200923012106-3a18fb78178f
```

Comment:
I _think_ the only thing keeping a dependency on `crossplane` around is the target controller. I know we are thinking we want to have `KubernetesApplication` deprecated for at least a release before ripping out, but I wonder how we feel about potentially modifying it to only support the case where you manually point it at a secret? That would allow us to remove all deps on `crossplane` in other repos. I would be happy to fix up the KA functionality and pull this out if y'all think this would be worth it. It should be relatively low effort. Thoughts? @negz @muvaf

Reply:
It might be as simple as just removing these controllers in providers and then updating documentation.

Reply:
I don't feel super strongly, but I feel like it would be less work overall to just leave these for now and remove them in the next release. Do we gain much by frontloading the work to this release?

Reply:
Just going ahead and breaking the dependency. Not a huge deal but if its all the same and doesn't detract from other efforts I may go ahead with it.